examples/jsm/controls/DragControls.js

```diff
@@ -75,14 +85,27 @@ class DragControls extends EventDispatcher {
 			if ( scope.enabled === false ) return;
 
 			updatePointer( event );
+			_end.copy( _pointer );
+			_diff.subVectors( _end, _start );
+			_start.copy( _end );
```

Comment:
I think this bit can be changed so we avoid unnecessary computations. The `_diff` computation should only be done inside the `scope.mode === 'rotate'` block. At the end of `onPointerDown()` and `onPointerMove()` if would do this:
```js
_previousPointer.copy( _pointer );
```
And `_diff` is then computed like so:
```js
_diff.subVectors( _pointer, _previousPointer );
```